app/models/liabilitycalculation/LiabilityCalculationResponse.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.liabilitycalculation
+
+import play.api.libs.json._
+
+sealed trait LiabilityCalculationResponseModel
+
+case class LiabilityCalculationError(status: Int, message: String) extends LiabilityCalculationResponseModel
+object LiabilityCalculationError {
+  implicit val format: OFormat[LiabilityCalculationError] = Json.format[LiabilityCalculationError]
+}
+
+case class LiabilityCalculationResponse(metadata: Metadata,
+                                        calculation: Option[Calculation]
+                                       ) extends LiabilityCalculationResponseModel
+object LiabilityCalculationResponse {
+  implicit val format: OFormat[LiabilityCalculationResponse] = Json.format[LiabilityCalculationResponse]
+}
+
+case class Metadata(calculationTimestamp: String, crystallised: Boolean)
```

Comment:
I would move the case class and the companion to independent file.

Reply:
In my view there is not sufficient complexity to warrant a separate file for this one (at the cost of maintenance overhead and lower readability)

Reply:
It is not about complexity. It is about clarity in the project structure. It is harder to find a class in a file that is named something different. Not being able to find something has more overhead to me. You can keep it as is if you prefer.